images/win/scripts/Installers/Install-PHP.ps1

```diff
@@ -11,7 +11,7 @@ Choco-Install -PackageName php -ArgumentList "--force", "--params", "/InstallDir
 Choco-Install -PackageName composer -ArgumentList "--ia", "/DEV=$installDir /PHP=$installDir"
 
 # update path to extensions and enable curl and mbstring extensions, and enable php openssl extensions.
-((Get-Content -path $installDir\php.ini -Raw) -replace ';extension=curl','extension=curl' -replace ';extension=mbstring','extension=mbstring' -replace ';extension_dir = "ext"','extension_dir = "ext"' -replace 'extension=";php_openssl.dll"','extension_dir = "php_openssl.dll"') | Set-Content -Path $installDir\php.ini
+((Get-Content -path $installDir\php.ini -Raw) -replace ';extension=curl','extension=curl' -replace ';extension=mbstring','extension=mbstring' -replace ';extension_dir = "ext"','extension_dir = "ext"' -replace ';extension=openssl','extension=openssl') | Set-Content -Path $installDir\php.ini
```

Comment:
Maybe I miss something, but:

Get-Content c:\tools\php\php.ini

1. Windows Server 2016
![image](https://user-images.githubusercontent.com/47745270/104582011-696cdc80-5670-11eb-9627-c3e77ad42208.png)

2. Windows Server 2019
![image](https://user-images.githubusercontent.com/47745270/104582072-80133380-5670-11eb-8270-2fa1a5d570cd.png)

Reply:
yes, but this script now shows all php extensions (curl, mbstring, openssl) that are enabled even if default php.ini will be changed sometimes.

Reply:
@al-cheb @LeonidLapshin Strangely on a clean VM with php installed by choco it's commented 🤔
![image](https://user-images.githubusercontent.com/48208649/104590815-e56d2180-567c-11eb-9658-ae8cb80dd4b2.png)